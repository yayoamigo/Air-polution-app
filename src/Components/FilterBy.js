import React from 'react'

export const FilterBy = () => {
  return (
    <select >
      <option className='' value=""> Filter by continent</option>
    </select>
  )
}
