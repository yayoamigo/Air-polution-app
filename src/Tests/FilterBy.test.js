import { Provider } from 'react-redux';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import FilterBy from '../Components/FilterBy';
import store from '../Redux/configureStore';

it('render', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <FilterBy />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
