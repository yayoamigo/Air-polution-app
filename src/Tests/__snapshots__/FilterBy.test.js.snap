// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div
  className="filterDiv"
>
  <p
    className="filterTitle"
  >
    Filter by continent
  </p>
  <select
    className="selectdiv"
    onChange={[Function]}
    value="All"
  >
    <option
      className="select"
      value="All"
    >
       All
    </option>
    <option
      className="select"
      value="Africa"
    >
       Africa
    </option>
    <option
      className="select"
      value="Europe"
    >
       Europe
    </option>
    <option
      className="select"
      value="North America"
    >
       North America
    </option>
    <option
      className="select"
      value="Asia"
    >
       Asia
    </option>
    <option
      className="select"
      value="South America"
    >
       South America
    </option>
    <option
      className="select"
      value="Oceania"
    >
       Oceania
    </option>
  </select>
</div>
`;
